collector/collector.go

```diff
@@ -28,23 +28,36 @@ import (
 	"time"
 
 	"github.com/google/certificate-transparency-monitor/client"
+	"github.com/google/certificate-transparency-monitor/ctlog"
 	"github.com/google/certificate-transparency-monitor/sthgetter"
 	"github.com/google/certificate-transparency-monitor/storage/print"
 )
 
 var (
 	sthGetPeriod = flag.Duration("sth_get_period", 1*time.Minute, "How regularly the monitor should get an STH from the Log")
 	logURL       = flag.String("log_url", "", "The URL of the Log to monitor, e.g. https://ct.googleapis.com/pilot/")
+	logName      = flag.String("log_name", "", "A short, snappy, canonical name for the Log to monitor, e.g. google_pilot")
+	b64PubKey    = flag.String("public_key", "", "The base64-encoded public key of the Log to monitor")
 )
 
 func main() {
 	flag.Parse()
 	if *logURL == "" {
 		log.Fatalf("No Log URL provided.")
 	}
+	if *logName == "" {
+		log.Fatalf("No Log name provided.")
```

Comment:
Might want to switch to glog like the other repos. If so these should probably all be glog.Exitf() in main.

Reply:
No particular rush for this but we've sort of standardized on glog in the other repos.